From: Georges Basile Stavracas Neto <georges.stavracas@gmail.com>
Date: Sun, 30 May 2021 21:27:51 -0300
Subject: dnd: Return Clutter.EVENT_STOP instead of true

Trivial cleanup.

Clutter.EVENT_STOP is more semantic, and this 'true' is inconsistent
with the 'return Clutter.EVENT_PROPAGATE;' a few lines above.

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/1867>

Origin: https://gitlab.gnome.org/GNOME/gnome-shell/-/commit/f0d8eb19
---
 js/ui/dnd.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/dnd.js b/js/ui/dnd.js
index 34ed8ae..252ebb8 100644
--- a/js/ui/dnd.js
+++ b/js/ui/dnd.js
@@ -518,7 +518,7 @@ var _Draggable = class _Draggable {
             }
         }
 
-        return true;
+        return Clutter.EVENT_STOP;
     }
 
     _pickTargetActor() {
